jitsi/020-deployment.yaml

```diff
@@ -0,0 +1,223 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: jitsi
+  namespace: jitsi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: jicofo
+    app.kubernetes.io/part-of: jitsi
+  name: jicofo
+  namespace: jitsi
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: jicofo
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: jicofo
+    spec:
+      containers:
+        - name: jicofo
+          image: jitsi/jicofo:stable-8922-1
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: XMPP_SERVER
+            value: prosody
+          - name: XMPP_DOMAIN
+            value: meet.jitsi
```

Comment:
I assume these are not correct?

Reply:
It's an internal domain, I think it should be fine to be `meet.jitsi`, though this is the default so I'm not sure it has to be specified.